Formula/xinit-privileged.rb

```diff
@@ -0,0 +1,45 @@
+class XinitPrivileged < Formula
+  desc "Launchd plist for xinit in privileged mode"
```

Comment:
Could this be made part of `xinit` formula?

Reply:
This is somewhat a hack because each formula can only have one service, but xinit provides two.

Reply:
Feels a bit weird and like a configuration option we could mention in `caveats` instead?

Reply:
Can you even configure `brew services` to use another plist?

Reply:
Yeh, there's a `--file` argument.

Reply:
Ah nice, that might work then.